pkgs/desktops/gnome-3/extensions/window-is-ready-remover/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "window-is-ready-remover";
+  version = "1.02";
+
+  src = fetchFromGitHub {
+    owner = "nunofarruca";
+    repo = "WindowIsReady_Remover";
+    rev = "a9f9b3a060a6ba8eec71332f39dc2569b6e93761";
+    sha256 = "0l6cg9kz2plbvsqhgwfajknzw9yv3mg9gxdbsk147gbh2arnp6v3";
+  };
+
+  uuid = "windowIsReady_Remover@nunofarruca@gmail.com";
+
+  installPhase = ''
+    mkdir -p $out/share/gnome-shell/extensions/
+    cp -r ${uuid} $out/share/gnome-shell/extensions/${uuid}
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Removes window is ready Notification";
+    homepage = "https://github.com/nunofarruca/WindowIsReady_Remover";
```

Comment:
No license means All rights reserved so we cannot provide it to users.

Reply:
[Licensing](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes). Set to `stdenv.lib.licenses.unfree`. Still doesn't have items such as platform and maintainers.